en/sim_gazebo_gz/tools_avl_automation.md

```diff
@@ -0,0 +1,141 @@
+# Advanced Lift Drag (AVL) Automation Tool
+
+This tool automates the writing of the Advanced Lift Drag plugin used by the Advanced Plane by automatizing the coefficient generation and requiring minimal user calculations.
+
+## Setup
+
+In order to run this tool, it is necessary to follow these steps:
+
+1. Download AVL 3.36 from <https://web.mit.edu/drela/Public/web/avl/>.
+   The file for AVL version 3.36 can be found about halfway down the page.
+2. After downloading, extract AVL and move it to the home directory using:
+
+```sh
```

Comment:
FYI only, I am starting to use `sh` to indicate "generic shell" through the docs. `shell` is probably even more correct, but I'd rather stick with `sh` and change it globally than have it mixed across the docs.